help/en/releasenotes/current-draft-note.shtml

```diff
@@ -344,12 +349,12 @@
 
         <h4>Uhlenbrock</h4>
             <ul>
-                <li></li>
+                <li> Extension with additional CVs for the Extended Function Mapping</li>
             </ul>
 
         <h4>Umelec</h4>
             <ul>
-                <li></li>
+                <li> </li>
```

Comment:
There is a space added on this line between `<li>` and `</li>`. This might confuse the script that reads this file and creates the release notes for the next release so I think that space needs to be removed.